oct/tests/deployment/deploy_app.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=no-self-use # pyATS-related exclusion
+# pylint: disable=no-member
+# pylint: disable=invalid-name
+# pylint: disable=too-many-branches
+import os
+from pyats.topology import Device, loader
+from pyats.aetest import Testcase, test, main
+from pyats.utils.fileutils import FileUtils
+from requests.exceptions import ConnectTimeout
+
+TWO_NON_EMPTY_LINES = 2
+
+
+def copy_file_to_server() -> None:
+    application = loader.load("testbed.yaml")
+    futils = FileUtils(testbed=application)
+    futils.copyfile(
+        source=f"{os.path.join(os.getcwd(), 'docker-compose.yaml')}",
+        destination=f"scp://{testbed.devices.vm.connections.main.ip}/home/adminaccount/oct",
+    )
+
+
+def is_deploy_app(device: Device) -> bool:
+    non_empty_lines_counter = 0
+    for iterator in device.execute("docker-compose ps -q"):
+        if iterator == "\n":
+            non_empty_lines_counter += 1
+    if non_empty_lines_counter == TWO_NON_EMPTY_LINES:
```

Comment:
@GoshkaEgoshka you don't need `if` here.
```suggestion
    return non_empty_lines_counter == 2:
```